2015-02-06 pangram review.md

```diff
@@ -5,16 +5,16 @@ Hello, CiO People!
 
 Today I would examine the fresh CheckiO mission ["Pangram"](http://www.checkio.org/mission/pangram/share/6d2c66db9c7d72144d48c01ae323e868/).
 This mission was based on an idea by [Sim0000](http://www.checkio.org/user/Sim0000/),
-which had added to a recent forum [post](http://www.checkio.org/forum/post/2977/mission-ideas/#comment-23718).
+which was added to a recent forum [post](http://www.checkio.org/forum/post/2977/mission-ideas/#comment-23718).
 You can propose youк own ideas there too and we will try to make them happen.
 
 [Pangram](https://en.wikipedia.org/wiki/Pangram) is a sentence which uses every letter of the alphabet at least once.
 In this mission you need to check text and verify if it is a pangram for the English alphabet or not.
 
 There are many methods which can be employed to solve this problem. 
-As we can see, it easily to write a solution with a time complexity of O(N), 
-you just use hash table data types to solve the problem. 
-However, we will not stop at this and will looking for more interesting solutions which were made by CheckiO players.
+As we can see, it's easy to write a solution with a time complexity of O(N), 
+you just have to use hash table data types to solve the problem. 
```

Comment:
"have to" change the meaning of the sentence. We propose one of the method, but user don't have to use hash tables.